### PR TITLE
r.mblend: update for GRASS 8

### PR DESCRIPTION
**Pull request by ldesousa**
* takes into account new behaviour in v.what.rast (inner buffer applied to region boundary)
* updates documentation with reference to new GitHub addons repository
* version number bumped to 1.2.0

### CODE REVIEW

File: src/raster/r.mblend/r.mblend.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def cleanup():
         gscript.run_command(
             "g.remove", type="all", name=TMP_MAPS.pop(), flags="f", quiet=True
         )
+    return None
```
Comment (petrasovaa) [inline src/raster/r.mblend/r.mblend.py]: This should not be needed.

**Comment by neteler** [comment]
Please consider to install `pre-commit` which catches black errors prior to submitting to GitHub (see https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md#use-pre-commit for an intro and setup).

File: src/raster/r.mblend/r.mblend.py (hunk 1 above)
Comment (ldesousa) [reply src/raster/r.mblend/r.mblend.py]: Correct with f186bd5a201623a7d624802bd23ea9319a665668

**Comment by ldesousa** [comment]
@neteler I suppose the `releasebranch_8_2` test is supposed to be failing after the new release. Correct?

**Review by neteler — APPROVED** [review]

**Comment by ldesousa** [comment]
@petrasovaa Can this PR be merged? Checks are passing and the changes are added.

**Pull request merged**